bundles/org.openhab.binding.deconz/src/main/resources/ESH-INF/thing/sensor-thing-types.xml

```diff
@@ -35,6 +36,14 @@
 		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS"/>
 	</channel-type>
 
+	<channel-type id="last_seen">
+		<item-type>DateTime</item-type>
+    <label>Last Seen</label>
```

Comment:
Sorry, there is still some wrong indention

Reply:
![image](https://user-images.githubusercontent.com/13451066/85835400-4e4e8400-b795-11ea-8012-072bf7915aa5.png)

Atom is weird... falling back to VI then